pkgs/development/python-modules/dm-tree/cmake.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/tree/CMakeLists.txt b/tree/CMakeLists.txt
+index 5e7eb43..7ad6d1a 100644
+--- a/tree/CMakeLists.txt
++++ b/tree/CMakeLists.txt
+@@ -50,62 +50,18 @@ if(APPLE)
+   set (CMAKE_FIND_FRAMEWORK LAST)
+ endif()
+ 
+-# Fetch pybind to be able to use pybind11_add_module symbol.
+-set(PYBIND_VER v2.6.2)
+-include(FetchContent)
+-FetchContent_Declare(
+-  pybind11
+-  GIT_REPOSITORY https://github.com/pybind/pybind11
+-  GIT_TAG        ${PYBIND_VER}
+-)
+-if(NOT pybind11_POPULATED)
+-    FetchContent_Populate(pybind11)
+-    add_subdirectory(${pybind11_SOURCE_DIR} ${pybind11_BINARY_DIR})
+-    include_directories(${pybind11_INCLUDE_DIR})
+-endif()
++find_package(pybind11 CONFIG REQUIRED)
+ 
+ # Needed to disable Abseil tests.
+ set (BUILD_TESTING OFF)
+ 
+ # Include abseil-cpp.
+-set(ABSEIL_VER 20210324.2)
+-include(ExternalProject)
+-ExternalProject_Add(abseil-cpp
+-  GIT_REPOSITORY    https://github.com/abseil/abseil-cpp.git
```

Comment:
Use of vendored source is an anti-pattern. Would be nice if upstream supported `find_package`

Reply:
not a blocker for this PR though

Reply:
Rebased to the most recent commit and changed the patch to hopefully be submittable to the upstream (= falling back to vendored deps if `find_package` fails), will try to open PR with them but keeping the patch here for now as I want to decouple upstream PR from this one.